### ISSUE TITLE
Consider allowing incremental dispvm writebacks

### ISSUE THREAD

**Issue by jpouellet** [QubesOS/qubes-issues#5132]
In https://groups.google.com/d/msg/qubes-users/GZflIR6Wnx4/xRKE3XNZBwAJ, airelemental wrote:
> b) Can't save progress. qvm-copy-to-vm only copies back the edits after the VM shuts down, right? So what if the system crashes in the meantime?

This is a good point. We may wish to consider allowing the open-in-dvm flow to write-back the file multiple times to the originating VM rather than only at shutdown. We could detect writes in the child DVM with an inotify watch, and would need to carefully think about the implications of allowing the file to be accepted back in the parent multiple times - there are perhaps some attacks enabled by this which don't immediately come to mind - but it seems a potentially worthwhile usability improvement if done safely.

**Comment by marmarek** [comment]
Good point, but indeed it needs to be done carefully. There is also possible consistency issue - reading the file while the applications is still running may be race - if user quickly save the file twice, we might end up picking up partial (the second) save.
